src/consumption/elevate.js

```
@@ -2061,6 +2075,106 @@ const publishProgram = function async(programData) {
+			if (userMappingResponse.status !== responseCode.ok) {
+				console.error('Error updating user extensions:', userMappingResponse)
+				throw new Error('Failed to update user extensions')
+			}
+
+			console.log('Successfully updated user extensions for program:', programId)
+			return resolve(true)
+		} catch (error) {
+			console.error('Error in createOrUpdateUserProgramMapping:', error)
+			return reject(error)
+		}
+	})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)